hrqb/base/target.py

```diff
@@ -0,0 +1,40 @@
+"""hrqb.base.target"""
+
+import json
+
+import luigi  # type: ignore[import-untyped]
+import pandas as pd
+from luigi.format import MixedUnicodeBytes  # type: ignore[import-untyped]
+
+PandasObject = pd.DataFrame | pd.Series
+
+
+class HRQBLocalTarget(luigi.LocalTarget):
+    """Target is local file with path and table name init."""
+
+    def __init__(self, path: str, table_name: str) -> None:
+        super().__init__(path, format=MixedUnicodeBytes)
+        self.path = path
+        self.table_name = table_name
```

Comment:
[Non-blocking] How will `table_name` attribute be used?

Reply:
That's a great question.  

I'm anticipating that these helper base classes may grow slightly to accomodate repeating patterns or edge cases in the Tasks as they get defined.  For Transform and Load Tasks, they _should_ be concerned with a single Quickbase table.  If that table name is defined, it could be used to programatically name files, or get a list of Tasks that deal with a particular table, etc.

But as of now, it's not directly used.  It's more of a nudge that a Task -- _except_ Extract Tasks which could arguably be just to get a lump of raw data -- should be oriented around a single Quickbase table.